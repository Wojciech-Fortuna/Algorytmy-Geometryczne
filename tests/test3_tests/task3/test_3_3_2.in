-0.0011310497406990389 0.05034364616169651
-0.03018346909553775 0.03686325400483377
-0.042159275547150654 0.0038362932205200373
-0.02641330780521517 -0.029190667563793703
0.0006431438076880541 -0.046041157759872144
0.027477821227042898 -0.029527677367715277
0.03235685348510742 0.001140214789147484
0.028586692194784838 0.02911202851463769