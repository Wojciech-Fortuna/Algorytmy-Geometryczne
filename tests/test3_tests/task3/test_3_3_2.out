1 7
2 7
2 6
3 6
3 5